### ISSUE TITLE
empty BEGIN OF END OF should give error

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1726]
```abap
TYPES: BEGIN OF language,
         END OF language.
```

merge with and rename rule https://rules.abaplint.org/begin_end_names/ ?

+ DATA + CONSTANTS + ENUM + CLASS-DATA etc

give a parser error? perhaps not, this is a structure error, where the error descriptions are best-as-say cryptic
